addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -784,3 +764,45 @@ a.no-decoration {
         border-radius: 50%;
     }
 }
+
+.o_wsale_share_btn {
+    &::after {
+        display: none;
+    }
+}
+
+.o_wsale_share_menu {
+    min-width: 3rem;
+}
+
+.o_total_border {
+    border: $border-width solid $border-color;
```

Comment:
🤔... 
```scss
    @include media-breakpoint-up(lg) {
        border: $border-width solid $border-color;
    }
 ```
 ?

Reply:
This has to be done to have no border on the card on screen < than lg for the  `short_cart_summary` and apply the border on the accordion of the `cart_summary` using the same class